functorch/_src/aot_autograd.py

```diff
@@ -55,6 +58,8 @@ def dynamo_timed(x):
 aten = torch.ops.aten
 
 
+KNOWN_TYPES = [torch.Tensor, int, str, float, bool, torch.SymInt, torch.SymFloat]
```

Comment:
you could use `[...] + py_sym_types` to be more future-proofed 

(https://github.com/pytorch/pytorch/blob/0d87a4fec89fc78e568224935897ec585a6368a6/torch/fx/experimental/proxy_tensor.py#L59)